app/elements/demo-tabs.html

```diff
@@ -97,22 +120,88 @@
        */
       name: {
         type: String
-      }
+      },
+
+      editorOpenFile: {
+        type: String,
+        value: 'index.html',
+      },
+
+      /**
+       * absolute path to the project directory
+       *
+       * e.g. /3.0/start/samples/custom-element
+       *     or /3.0/start/samples/employee-list
+       */
+        projectPath: {
+        type: String,
+      },
+
+      _displaySelected: {
+        type: String,
+        value: 'static',
+      },
+
+      _isPlunkr: {
+        type: Boolean,
+      },
+
+      _isStackBlitz: {
+        type: Boolean,
+      },
     },
 
+    observers: ['_srcProjectPathChanged(src, projectPath)'],
+
     attached: function() {
       this._selected = 0;
     },
 
-    _hidePlunkrButton: function(src) {
-      return (!src || src === '');
+    _srcProjectPathChanged: function(src, projectPath) {
+      this._srcDefined = !!this.src;
+      this._projectPathDefined = !!this.projectPath;
+
+      if (!this._srcDefined && !this._projectPathDefined) {
+        this._hideEditorButton = true;
+      }
     },
 
-    // Call out to a global method defined by app.js
-    _plunkerLaunched: function() {
+    _hideEditorButton: function(srcDefined, prjoectPathDefined) {
```

Comment:
`_hideEditorButton` is a method AND a property?

Reply:
oops